charts/orchestrator-k8s/values.yaml

```diff
@@ -189,8 +206,12 @@ backstage:
           providers:
             github:
               development:
-                clientId: "INSERT VALID CLIENT ID HERE"
-                clientSecret: "INSERT VALID CLIENT SECRET HERE"
+                clientId: ${GH_PROVIDER_CLIENT_ID}
```

Comment:
Is env-var valid in helm chart values file?

Reply:
Well...
```
2024-07-25T14:15:16.540369739Z stderr F ForwardedError: Plugin 'auth' startup failed; caused by Error: Failed to initialize github auth provider, Missing required config value at 'auth.providers.github.development.clientId' in 'app-config-from-configmap.yaml'
```

Reply:
I wanted to push 2 more updates but wasn't fast enough to hold the pr.
anyhow, even with new versions rhdh still fail with MODULE_NOT_FOUND, the main problem remains...